app/src/lib/shells/win32.ts

```diff
@@ -114,52 +114,35 @@ export async function getAvailableShells(): Promise<
   return shells
 }
 
-function addErrorTracing(context: string, cp: ChildProcess) {
-  cp.stderr.on('data', chunk => {
-    const text = chunk instanceof Buffer ? chunk.toString() : chunk
-    log.debug(`[${context}] stderr: '${text}'`)
-  })
-
-  cp.on('exit', code => {
-    if (code !== 0) {
-      log.debug(`[${context}] exit code: ${code}`)
-    }
-  })
-}
-
-export async function launch(
+export function launch(
   foundShell: IFoundShell<Shell>,
   path: string
-): Promise<void> {
+): ChildProcess {
   const shell = foundShell.shell
 
   if (shell === Shell.PowerShell) {
     const psCommand = `"Set-Location -LiteralPath '${path}'"`
-    const cp = spawn(
-      'START',
-      ['powershell', '-NoExit', '-Command', psCommand],
-      {
-        shell: true,
-        cwd: path,
-      }
-    )
-    addErrorTracing(`PowerShell`, cp)
+    return spawn('START', ['powershell', '-NoExit', '-Command', psCommand], {
```

Comment:
I notice you are using `log.info` to give some info about other shells. Any reason not to do it for PowerShell?

Reply:
@iAmWillShepherd there's no "path" here to log